article/app/views/package.scala

```diff
@@ -36,6 +38,10 @@ object MainCleaner {
 object BodyCleaner {
   def apply(article: Article, html: String, amp: Boolean)(implicit request: RequestHeader) = {
     implicit val edition = Edition(request)
+
+    val shouldShowAds = !article.content.shouldHideAdverts && article.metadata.section != "childrens-books-site"
```

Comment:
@JonNorman That's happening client side but on amp page ads are included server side during page generation.